content/en/docs/concepts/storage/volumes.md

```diff
@@ -11,228 +11,193 @@ weight: 10
-For any kind of volume in a given pod, data is preserved across container restarts.
-
-At its core, a volume is a directory, possibly with some data in it, which
-is accessible to the containers in a pod. How that directory comes to be, the
-medium that backs it, and the contents of it are determined by the particular
-volume type used.
-
-To use a volume, specify the volumes to provide for the Pod in `.spec.volumes`
-and declare where to mount those volumes into containers in `.spec.containers[*].volumeMounts`.
-A process in a container sees a filesystem view composed from their Docker
-image and volumes. The [Docker image](https://docs.docker.com/userguide/dockerimages/)
-is at the root of the filesystem hierarchy. Volumes mount at the specified paths within
-the image. Volumes can not mount onto other volumes or have hard links to
-other volumes. Each Container in the Pod's configuration must independently specify where to
-mount each volume.
-
-## Types of Volumes {#volume-types}
-
-Kubernetes supports several types of volumes.
-
-### awsElasticBlockStore {#awselasticblockstore}
-
-An `awsElasticBlockStore` volume mounts an Amazon Web Services (AWS)
-[EBS volume](https://aws.amazon.com/ebs/) into your pod. Unlike
-`emptyDir`, which is erased when a pod is removed, the contents of an EBS
-volume are persisted and the volume is unmounted. This means that an
-EBS volume can be pre-populated with data, and that data can be shared between pods.
-
-{{< note >}}
-You must create an EBS volume by using `aws ec2 create-volume` or the AWS API before you can use it.
-{{< /note >}}
-
-There are some restrictions when using an `awsElasticBlockStore` volume:
+## Introduction
 
-* the nodes on which pods are running must be AWS EC2 instances
-* those instances need to be in the same region and availability zone as the EBS volume
-* EBS only supports a single EC2 instance mounting a volume
+By default, everything with the filesystems of a container is ephemeral. During startup, each pod in a container gets a filesystem created based on the layers in its {{< glossary_tooltip text="container image" term_id="image" >}}. When a container is restarted or destroyed, that root filesystem, including any modifications to it, is lost.
 
-#### Creating an AWS EBS volume
+A Kubernetes volume's lifecycle is tied to the *pod* that encloses it. The pod maintains the volume as different containers are started or stopped. Multiple volumes of any type may be connected to a pod. The [PersistentVolume](/docs/concepts/storage/persistent-volumes/) abstraction builds on the volume concept to let pods access storage that outlasts the lifetime of a pod.
 
-Before you can use an EBS volume with a pod, you need to create it.
+One use case for volumes is providing directories to pods, possibly with some data in it.  How that directory comes to be, the medium that backs it, and the contents of it are determined by the particular volume type used.
```

Comment:
I think the statement about "providing a filesystem to pods" is pretty basic and should come before the lifecycle part.

Reply:
Ok, I reorganized some paragraphs to address this.